### ISSUE TITLE
TFlite key point detection reasoning problem

### ISSUE THREAD

**Issue by jiokul** [ultralytics/ultralytics#20671]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

I will get the TFlite model from the yolo training model through export, but the confidence detected on the Android side is very low, far less than 0.1. I also export the official pre-training model to TFLite, and I can get the correct result in the same reasoning script. What is the possible reason?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @jiokul, thank you for reaching out and using Ultralytics 🚀! This is an automated response to help you get started—an Ultralytics engineer will assist you soon.

We recommend visiting the [Docs](https://docs.ultralytics.com/) where you’ll find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples, as well as answers to common export and inference questions.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) (MRE) including:
- The exact export command and model used
- Sample input data
- The inference script and output logs from your Android side

This will help us investigate the TFLite confidence issue more efficiently.

For custom training or export questions, include as much detail as possible—your training/export commands, model details, and results—so the team can provide targeted guidance. Be sure to review our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) too.

Join the Ultralytics community to connect and get support:
- Real-time chat: [Discord](https://discord.com/invite/ultralytics) 🎧
- In-depth discussions: [Discourse](https://community.ultralytics.com/)
- Community threads: [Subreddit](https://reddit.com/r/Ultralytics)

## Upgrade

Please ensure you are using the latest `ultralytics` package version and all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Environments

YOLO can be run in these up-to-date verified environments (with dependencies like [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/), and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM: [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI: [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**: [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

Thank you for your patience and for contributing to Ultralytics!

**Comment by Y-T-G** [comment]
Did you test the TFLite model in Ultralytics?
